packages/composite-checkout-wpcom/src/components/wp-contact-form.js

```diff
@@ -531,8 +368,64 @@ function joinNonEmptyValues( joinString, ...values ) {
 	return values.filter( value => value.length > 0 ).join( joinString );
 }
 
-const StateSelectWrapper = styled.div`
-	& select {
-		width: 100%;
+function contactDetailsFormat( { isDomainFieldsVisible } ) {
```

Comment:
Minor, but might this be more readable with a verb like `getContactDetailsFormat`? As it is I first thought this was supposed to format the contact details.